### PR TITLE
Twinkle Star: Version 2.010; ttfautohint (v1.8.3) added

### PR DESCRIPTION
**Pull request by vv-monsalve**
b6248ca: [gftools-packager] Twinkle Star: Version 2.010; ttfautohint (v1.8.3) added

* Twinkle Star Version 2.010; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/googlefonts/twinkle-star at commit https://github.com/googlefonts/twinkle-star/commit/f850cc0250ac33e9797cfa1318a670a055f2b794.

 ba9793d: [gftools-packager] ofl/twinklestar remove METADATA "source".  google/fonts#2587

### CODE REVIEW

**Comment by vv-monsalve** [comment]
Part of Batch 4 of TypeSetIT fonts

File: ofl/twinklestar/DESCRIPTION.en_us.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<p>
+Twinkle Star is a cute and fun juvenile script. It comes in two styles: A Script style with a more curvy quality and a casual style with more children writing resemblance.
```
Comment (RosaWagner) [inline ofl/twinklestar/DESCRIPTION.en_us.html:2]: Is there still 2 styles? Which one is this one?

Comment (vv-monsalve) [reply ofl/twinklestar/DESCRIPTION.en_us.html:2]: He means within the font (stylistic set) but perhaps it's not clear enough. I'll change it.

**Comment by vv-monsalve** [comment]
Updated

## Twinkle Star: Version 2.010; ttfautohint (v1.8.3) added

---
 633e36a: [gftools-packager] Twinkle Star: Version 2.010; ttfautohint (v1.8.3) added

* Twinkle Star Version 2.010; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/googlefonts/twinkle-star at commit https://github.com/googlefonts/twinkle-star/commit/b91b9bb130e238e2ab91d1dfd855cfcbb74c5677.

 3a92e39: [gftools-packager] ofl/twinklestar remove METADATA "source".  google/fonts#2587

File: ofl/twinklestar/DESCRIPTION.en_us.html (hunk 1 above)
Comment (vv-monsalve) [reply ofl/twinklestar/DESCRIPTION.en_us.html]: Updated now

**Pull request merged**